ui/components/app/dropdowns/network-dropdown.js

```diff
@@ -130,13 +127,11 @@ class NetworkDropdown extends Component {
         <Button
           type="secondary"
           onClick={() => {
-            if (this.props.addPopularNetworkFeatureToggledOn) {
-              this.props.history.push(ADD_POPULAR_CUSTOM_NETWORK);
-            } else {
-              getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
-                ? global.platform.openExtensionInBrowser(ADD_NETWORK_ROUTE)
-                : this.props.history.push(ADD_NETWORK_ROUTE);
-            }
+            getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
+              ? global.platform.openExtensionInBrowser(
+                  ADD_POPULAR_CUSTOM_NETWORK,
```

Comment:
were these later commits meant to be in this PR? 
https://github.com/MetaMask/metamask-extension/pull/15558/commits/b0844e4034181903974b8b17572a49e75d885afc

Reply:
Fixed, thank you!